src/App.tsx

```diff
@@ -180,33 +162,41 @@ function App() {
   useEffect(() => {
     async function callToAPIs() {
       try {
-        const [movieResponse, selMovieResponse] = await Promise.all([
+        const [movieResponse] = await Promise.all([
           getMovies(state.search, state.page, mediaType),
           getSelected(selectedID),
```

Comment:
Can I remove the getSelected func call from here if I'm calling it in the other useEffect?

Reply:
whoops yeah definitely remove it